src/main/resources/application.yaml

```diff
@@ -54,7 +54,7 @@ idam:
     totp_secret: ${MANAGE_CASE_S2S_KEY:AAAAAAAAAAAAAAAA}
     url: ${S2S_URL:http://localhost:4502}
   s2s-authorised:
-    services: ${MANAGE_CASE_S2S_AUTHORISED_SERVICES:xui_webapp}
+    services: ${MANAGE_CASE_S2S_AUTHORISED_SERVICES:xui_webapp,ccd_data}
 oidc:
```

Comment:
I am not use but MANAGE_CASE_S2S_AUTHORISED_SERVICES is also defined in values.yaml, at the moment has one value. If  the env_variable is present it will only have   xui_webapp.  if not it will be xui_webapp,ccd_data.
It is the expected functionality ?